src/modules/main/types/experts.ts

```diff
@@ -0,0 +1,39 @@
+export interface IExperts {
+    userId: number,
+    emailId: string,
+    password: string,
+    status: ExpertStatus,
+    firstName: string,
+    secondName: string,
+    qualification: string,
+    phone: string,
+    bio: string,
+    createdAt: Date,
+}
+
+enum ExpertStatus {
+    'new',
+  'active',
+  'deleted'
+}
+
+export interface IExpertInstitution{
+    userId: number,
+    institutionId: number,
+    isPrimary: boolean
+}
+
+export interface IExpertCategory {
+    userId: number,
+    categoryId: number
+}
+
+export interface IExpertSource {
+    userId: number,
+    sourceId: number
+}
+
+export interface IExpertRole {
```

Comment:
we don't have the mapping as there's on the db.
in this example, it should probably be as another property on the IExpert